src/pugixml.hpp

```diff
@@ -91,8 +91,14 @@
 #endif
 #ifdef PUGIXML_HAS_NOEXCEPT
 #	define PUGIXML_NOEXCEPT noexcept
+#	ifdef PUGIXML_COMPACT
+#		define PUGIXML_NOEXCEPT_IF_NOT_COMPACT noexcept(false)
```

Comment:
Is noexcept(false) necessary here, as opposed to just not specifying noexcept?

Reply:
It is not necessary, if you prefer omitting it then I'll change it.
I did it this way from a "better explicit than implict" POV.

Reply:
Ok - please remove it if this is not necessary.

Reply:
Done